### DISCUSSION TITLE
咨询一下语音消息支持16000的amr嘛

### DISCUSSION THREAD

**Discussion by zmdyy0318** [mamoe/mirai#878] [Q&A] [answered]
版本：
全套2.0.0
mirai-api-http:1.9.6
场景：
自行生成amr语音，通过cpp的http上传。经过测试，amr如果是8000的话，QQ可以正常播放，但声音沉闷。amr如果是16000的话，QQ可以正常识别语音长度，但无法播放，语音再长也是瞬间结束。
问题：
是mirai目前只支持8000还是我的操作方法有问题。
16000语音附件：
文件头是#!AMR，本地可以正常播放。
[1611148220.amr.zip](https://github.com/mamoe/mirai/files/5842639/1611148220.amr.zip)

**Comment by Him188** [comment, marked as answer]
https://github.com/mamoe/mirai/issues/577

**Reply by zmdyy0318** [reply]
谢谢大佬回复，不会java的没太看懂Highway是什么。。。
